d2pio-jna/src/main/java/org/concord/sensor/d2pio/jna/D2PIOLibrary.java

```diff
@@ -189,7 +189,12 @@ private static String getNativeLibraryResourcePath() {
 				String arch = System.getProperty("os.arch").toLowerCase();
 				String osPrefix;
 				if (Platform.isWindows()) {
-					osPrefix = "Win7_Win32";
+					if (arch.indexOf("64") != -1) {
+						osPrefix = "Win7_Win64";
+					}
+					else {
+						osPrefix = "Win7_Win32";
+					}
```

Comment:
I feel like I remember seeing this code before. I share your confusion over where it might have gone.